### ISSUE TITLE
[tune] Stack Traces with Function API are really hard to parse

### ISSUE THREAD

**Issue by richardliaw** [ray-project/ray#14162]
### What is the problem?

https://discuss.ray.io/t/attributeerror-nonetype-object-has-no-attribute-get/918/2

We end up posting a dozen stack traces from various parts of the system; we should really make sure to isolate so that we minimize the number of stacktraces that we show.

**Comment by Yard1** [comment]
@xwjiang2010 has this been taken care of by your simplify stack traces PR?

**Comment by cszhu** [comment]
Hello! This P2 issue has seen no activity in at least 2 years. It will be closed in 2 weeks as part of ongoing cleanup efforts.

Please remove the `pending-cleanup` label if you believe this issue should remain open.

Thanks for contributing to Ray!
